simple-test-cases/openmp_reduction1.c

```diff
@@ -0,0 +1,25 @@
+///TAFFO_TEST_ARGS -fopenmp -Xvra -unroll=100
+#include <stdio.h>
+
+#define N (100)
+
+int main(int argc, char *argv[])
+{
+  float result __attribute__((annotate("target('result') scalar()"))) = 0.0;
+  float container[N] __attribute__((annotate("target('container') scalar()")));
+  float container_result __attribute__((annotate("target('container_result') scalar()"))) = 0;
+
+  int i = 0;
+
+#pragma omp parallel for reduction(+:result)
```

Comment:
```suggestion
  #pragma omp parallel for reduction(+:result)
```
It's more legible if you indent the openmp pragmas. Apply this everywhere.